From e326b944f204051077f16388694eaaa6d8daa047 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 21 Nov 2019 16:52:11 +0200
Subject: [PATCH] [SW-6506] hl: ppboot: load preboot according from new flash offset

Since U-boot, when serveing FW loader, does not aware to preboot new
location, we put need to put it elsewhere.
We choose the end of the u-boot chunk which is anyway just a pad since
chunk is 1MB

Change-Id: Idd08ff38ce3b4ebabaf28dedc992e29b11e6cec9
---

diff --git a/include/preboot.h b/include/preboot.h
index 1175ca0..7b03773 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -31,11 +31,11 @@
 #define SPL_SIZE		0x10000		/* 64KB */
 #define HL_UART_BAUDRATE	115200
 #define HL_SPI_FLASH_MAX_FREQ	25000000
-#define HL_SPI_PREBOOT_OFF	0x50000
 #define HL_SPI_NEXT_IMAGE_OFF	0x60000
 #define HL_SPI_MODIFI_IMAGE_OFF	0x60004
 #define HL_SPI_SPL_OFF		0xA0000
 #define HL_SPI_UBOOT_OFF	0xB0000
+#define HL_SPI_PREBOOT_OFF	0x1A0000
 #define HL_SPI_PRIMARY_BASE	0
 #define HL_SPI_SECONDARY_BASE	0x800000
 
